daru.gemspec

```diff
@@ -54,6 +54,8 @@ EOF
 
   # it is required by NMatrix, yet we want to specify clearly which minimal version is OK
   spec.add_runtime_dependency 'packable', '~> 1.3.9'
+  # for integrating yard-junk with Cl
+  spec.add_runtime_dependency 'yard-junk', '~>0.0.7'
```

Comment:
`yard-junk` is a `development_dependency` rather than a `runtime_dependency`, along with `rubocop` and `rspec`